server/controllers/ArticleController.js

```diff
@@ -119,6 +119,53 @@ class ArticleController {
       });
     });
   }
+
+  /**
+   * @description This method add new comment on an article
+     * @param {*} req
+     * @param {*} res
+     * @returns {object} returns an object of comment
+   */
+  static addCommentOnArticle(req, res) {
+    const { body } = req.body;
+    const { articleId } = req.params;
+    const userId = req.userData.id;
+    return Comment
+      .create({
+        body,
+        userId,
+        articleId
+      })
+      .then((comment) => {
+        const {
+          createdAt,
+          updatedAt
+        } = comment;
+        User.findByPk(userId)
+          .then((user) => {
+            const { fullName, avatarUrl } = user;
+            res.status(201).json({
+              status: 'success',
+              message: 'Comment added successfully',
+              comment: {
+                body: comment.body,
+                articleId: comment.articleId,
+                createdAt,
+                updatedAt,
+                commentBy: {
+                  fullName,
+                  avatarUrl
+                },
+              }
+            });
+          });
+      })
+      .catch(err => res.status(500).json({
+        errors: {
+          message: [err.message]
+        }
+      }));
+  }
```

Comment:
This looks okay @sulenchy, but I'd like to suggest we have comments in a different controller to separate concerns.

Reply:
Thanks, @mrSerious feedback acknowledged